src/views/view.h

```diff
@@ -120,8 +120,8 @@ typedef struct dt_view_t
   GSList *accel_closures;
 } dt_view_t;
 
-typedef enum dt_view_image_over_t
-{
```

Comment:
this change is not ok, dt style is to have curly-brace on a new line.

Reply:
Well, I thought so, but I tried and clang-format refused my commit.

Reply:
If I change to the intended style, viz. with the brace on the following line, an then I try to commit, this happens
````
$ git commit -a
Commit did not match clang-format
````

Running `git clang-format-3.8 --diff` on the command line, I get the following.
````
$ git clang-format-3.8 --diff
diff --git a/src/views/view.h b/src/views/view.h
index bb1afb7c5..acb63293f 100644
--- a/src/views/view.h
+++ b/src/views/view.h
@@ -120,8 +120,7 @@ typedef struct dt_view_t
   GSList *accel_closures;
 } dt_view_t;
 
-typedef enum dt_view_image_over_t
-{
+typedef enum dt_view_image_over_t {
   DT_VIEW_ERR = -1,
   DT_VIEW_DESERT = 0,
   DT_VIEW_STAR_1 = 1,
````

Should I update something? Or should I force the commit?

Reply:
Force push will be better. Thanks